app/components/task_list_item_references_component.rb

```diff
@@ -1,8 +1,10 @@
 class TaskListItemReferencesComponent < ViewComponent::Base
   include ViewHelper
 
-  def initialize(references:)
+  def initialize(references:, text: nil, path: nil)
     @references = references
+    @text = text
+    @path = path
```

Comment:
Given this component isn't used anywhere else (and so there's no need to customise the link text or path), I'd suggest keeping this simple for now and hardcoding them in the template rather than accepting parameters.

Reply:
(can always adopt the approach you've taken here if we need to display references in this way elsewhere in the sytem)